src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -118,7 +129,15 @@ class ExplorerButtonGroup extends React.Component {
       filter.data_format = this.props.filter.data_format;
     }
     let resultManifest = await this.props.downloadRawDataByTypeAndFilter(
-      resourceType, filter, [refFieldInResourceIndex, resourceFieldInResourceIndex],
+      resourceType,
+      filter,
+      [
+        refFieldInResourceIndex,
+        resourceFieldInResourceIndex,
+        fileNameField,
```

Comment:
if there's no `file_name` or `file_size` field in ES index, will this request fail?

Reply:
I don't know -- I'll test it now and find out.

Reply:
Yes, the request fails, and causes an unhandled Promise rejection, so the manifest is not downloaded.

Reply:
Update: After talking with @qingyashu we've decided to pre-check the hard-coded file manifest field names against the field names in Guppy's file index. This lets us safely request the new field names without having to make portal config changes.

Reply:
@qingyashu Let me know if the new approach makes sense -- I just wrapped the Guppy call in a try/catch block, and retry the request without the additional fields if it fails. (I experimented with pre-requesting the fields or using the allFields prop from the Guppy wrapper but this seemed simplest)